src/RichTextEditor/index.js

```diff
@@ -58,6 +58,19 @@ const RichTextEditor = (props) => {
   const { isEditable, canBeFormatted } = props;
 
   const onKeyDown = useCallback((event) => {
+    if (isHotkey(ENTER, event)) {
+      // eslint-disable-next-line no-restricted-syntax
+      for (const [n] of Node.ancestors(editor, editor.selection.focus.path, { reverse: true })) {
```

Comment:
@DianaLease this goes from the bottom up in the tree: https://docs.slatejs.org/api/nodes#static-methods